meinberlin/apps/plans/exports.py

```diff
@@ -48,11 +49,19 @@ def get_virtual_fields(self, virtual):
         virtual = super().get_virtual_fields(virtual)
         virtual['projects'] = ugettext('Projects')
         virtual['projects_links'] = ugettext('Project Links')
+        virtual['created'] = ugettext('Created')
+        virtual['modified'] = ugettext('Modified')
         return virtual
 
     def get_organisation_data(self, item):
         return item.organisation.name
 
+    def get_created_data(self, item):
```

Comment:
If we add the fields to the end of the virtual fields, it also makes more sense to me to add these methods to the end of the method list. Even though, it doesn't change anything about the function of it. So, if you think, they make more sense here, leave them here.